lib/swaggerValidator/trafficValidator.ts

```diff
@@ -146,7 +145,9 @@ export class TrafficValidator {
         spec = await this.loader.load(pathResolve(swaggerPath));
```

Comment:
Can you remove `pathResolve` here as dup?

Reply:
this is from liveValidator.swaggerList, different from `specPath`